src/libs/actions/Policy.ts

```diff
@@ -175,33 +179,33 @@ function deleteWorkspace(policyID, reports, policyName) {
                 errors: null,
             },
         },
-        ..._.map(reports, ({reportID}) => ({
+        ...reports.map<OnyxUpdate>(({reportID}) => ({
```

Comment:
Are these `<OnyxUpdate>` necessary?

Reply:
These are necessary, the type isn't inferred from the returned map value and without this it will result in a typescript error.